releases/release-4.0.16.md

```diff
@@ -0,0 +1,108 @@
+    - Fix the issue that optimistic transaction conflicts might cause transactions blocking each other [#11148](https://github.com/tikv/tikv/issues/11148)
+    - Fix the issue of incomplete log information from the `auto analyz` result [#29188](https://github.com/pingcap/tidb/issues/29188)
+    - Fix an issue that `NO_ZERO_IN_DATE` does not work on the default values [#26766](https://github.com/pingcap/tidb/issues/26766)
+    - Fix the issue that the Coprocessor Cache panel in Grafana does not display metrics. Now, Grafana displays the number of `hits`/`miss`/`evict`. [#26338](https://github.com/pingcap/tidb/issues/26338)
+    - Fix the issue that concurrently truncating the same partition hangs DDL [#26229](https://github.com/pingcap/tidb/issues/26229)
+    - Fix the issue that when the `CONCAT` function has a negative float type argument, the last digit of the argument number is chopped in the result  [#29417](https://github.com/pingcap/tidb/issues/29417)
+    - Fix the issue of an extra column in the query result when `NATURAL JOIN` is used to join multiple tables [#29481](https://github.com/pingcap/tidb/issues/29481)
+    - Fix the issue that `TopN` is wrongly pushed down to `indexPlan` when `IndexScan` is using a prefix index [#29711](https://github.com/pingcap/tidb/issues/29711)
+    - Fix the issue that retrying transactions with `DOUBLE` type auto-increment columns causes data error [#29892](https://github.com/pingcap/tidb/issues/29892)
++ TiKV
+
+    - Fix panic in rare conditions when merge, conf change and snapshot happen at the same time [#11475](https://github.com/tikv/tikv/issues/11475)
+    - Make negative sign as false when decimal divide result is zero [#29586](https://github.com/pingcap/tidb/issues/29586)
+    - Fix incorrect by-instance gRPC average duration [#11299](https://github.com/tikv/tikv/issues/11299)
+    - Fix CDC panic due to missing downstream [#11123](https://github.com/tikv/tikv/issues/11123)
+    - (dup) Fix the issue that the Raft connection is broken when the channel is full [#11047](https://github.com/tikv/tikv/issues/11047)
+    - Fix Max/Min bug when comparing signed and unsigned int64 [#10158](https://github.com/tikv/tikv/issues/10158)
+    - (dup) Fix the issue that CDC adds scan retries frequently due to the Congest error [#11082](https://github.com/tikv/tikv/issues/11082)
+
++ PD
+
+    - Fix panic issue after TiKV node scales in [#4344](https://github.com/tikv/pd/issues/4344)
+    - (dup) Fix slow leader election caused by stucked region syncer [#3936](https://github.com/tikv/pd/issues/3936)
+    - (dup) Fix the issue that evict-leader might leave leaders when the cluster has down peers [#4093](https://github.com/tikv/pd/issues/4093)
+
++ TiFlash
+
+    - Fix the issue that comparison between Decimal may cause overflow and report `Can't compare`
+    - Fix the issue that TiFlash fails to start up under platform without library `nsl`
```

Comment:
```suggestion
    - Fix the issue that comparison between Decimal may cause overflow and report `Can't compare`
    - Fix the issue that TiFlash fails to start up on some platforms due to the absence of library `nsl`
```

Reply:
There is a similar bug fix in 5.3.0 release notes: Fix the possible Can't compare error when comparing Decimal types. Are they the same?